forge-game/src/main/java/forge/game/trigger/TriggerExiled.java

```diff
@@ -84,7 +84,7 @@ public final boolean performTest(final Map<AbilityKey, Object> runParams) {
             return false;
         }
 
-        if (hasParam("Madness")) {
+        if (isKeyword(Keyword.MADNESS)) {
             if (cause == null || !cause.isKeyword(Keyword.MADNESS)) {
```

Comment:
the check here could probably use some more accuracy:
currently multiple _Falkenrath Gorger_ cause more than one trigger, but only the one doing the exiling should apply

Reply:
They are not the same "ctb", but part of the same keyword. So checking if they are the same keyword should work

Reply:
Self should check same host, but your way might be even safer

Reply:
doesn't work, because TriggerExiled is on the new Card object, so the Keyword Instance is different (and the Id of the Trigger is different too)

Maybe TriggerExiled should use LKI?

Reply:
that seems wrong / not worth the trouble
can `kw.getStatic()` be a match?

Reply:
i added getStatic check

what i find curious is that the two Madness didn't asked me to choose the ReplacementEffect 
(can you help debug this?)

Reply:
hm i need to test it with intrinsic Madness too

Reply:
well looks like that doesn't work either, the description for the RE is too hardcoded

Reply:
I think for intrinsic the logic might still need a small change
either both are null or equal?